VTEX - Orders API.json

```diff
@@ -3173,815 +3174,1437 @@
+                                                    "userName": {
+                                                        "type": "string",
+                                                        "description": "Call center operator user name."
+                                                    }
+                                                }
+                                            },
+                                            "followUpEmail": {
+                                                "type": "string",
+                                                "description": "Email of the store employee responsible for managing the order."
+                                            },
+                                            "lastMessage": {
+                                                "type": "string",
+                                                "description": "Last transactional message sent to the customer.",
+                                                "nullable": true
+                                            },
+                                            "hostname": {
+                                                "type": "string",
+                                                "description": "Main [account name](https://help.vtex.com/en/tutorial/what-is-an-account-name--i0mIGLcg3QyEy8OCicEoC)."
+                                            },
+                                            "invoiceData": {
+                                                "$ref": "#/components/schemas/InvoiceDataUpdated"
+                                            },
+                                            "changesAttachment": {
+                                                "$ref": "#/components/schemas/ChangesAttachmentUpdated"
+                                            },
+                                            "openTextField": {
+                                                "type": "object",
+                                                "description": "Optional field for additional information.",
+                                                "nullable": true,
+                                                "additionalProperties": {
+                                                    "type": "string",
+                                                    "description": "Additional information."
+                                                }
+                                            },
+                                            "roundingError": {
+                                                "type": "integer",
+                                                "description": "Rounding error total value in cents, when applicable. For example, in orders with a discount over non-integer multiplier items, the rounding price is performed per item, not after the sum of all items. That can cause a difference in the total discount value, which is informed in this field."
```

Comment:
> total discount value, which is informed in this field

What does this mean? Isn't this field the rounding error? I suggest removing this bit or naming the field explicitly instead of using "this field".